src/main/java/taxi/model/Car.java

```diff
@@ -0,0 +1,48 @@
+package taxi.model;
+
+import java.util.List;
+
+public class Car {
+    private long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
```

Comment:
будь уважним з чеклістом:
```suggestion
    public Car(String model, Manufacturer manufacturer) {
        this.model = model;
        this.manufacturer = manufacturer;
        drivers = new ArrayList<>();
```

Reply:
changed